usr/lib/anon-shared-helper-scripts/tor_enabled_check

```diff
@@ -23,40 +23,24 @@ check_tor_enabled_do() {
       fi
    fi
 
-   local line file_name file_name_list i
-
    shopt -s globstar
    shopt -s nullglob
+   shopt -s nocasematch
 
-   if [ -f /usr/share/tor/tor-service-defaults-torrc ]; then
-      file_name_list+="/usr/share/tor/tor-service-defaults-torrc"
-      file_name_list+=" "
-   fi
-   if [ -f /etc/tor/torrc ]; then
-      file_name_list+="/etc/tor/torrc"
-      file_name_list+=" "
-   fi
+   ## many function calls in this script are provided by torrc-parser
+   source /usr/lib/anon-gw-anonymizer-config/torrc-parser
+
+   ## $tor_start_command is set by this call
+   get_tor_start_command
+
+   ## extract default_torrc file and -f torrc file
+   extra_specified_torrc
+
+   ## call parser() to parse torrc files, arguments order matters
+   ## $torrc_all_in_one is set by the call
+   parser "$default_torrc" "$f_torrc"
```

Comment:
The use of variables `"$default_torrc" "$f_torrc"` seems surprising. Could you please document how these get set?

It's probably like the following?

```
## $tor_start_command is set by this call
## sets: default_torrc
## sets: f_torrc
get_tor_start_command
```

Reply:
Fixed in https://github.com/Whonix/anon-shared-helper-scripts/pull/3/commits/868fa53fb056fb9aa0e8483c59c897d129ac8036

Reply:
Btw the style is debateable.

* `function_call_a` vs `function_call_a()`
* `variable_name` vs `$variable_name`

`function_call_a()` may be not that create because in bash one can also write `function function_call_a`. Perhaps it should be `function_call_a()` or `function function_call_a` depending on the actual style of the person writing the script?
* `variable_name` may be better than `$variable_name` since bash also supports arrays `variable_a=x ; variable_b[$variable_a]=y`. (not often used, complex, not pretty imo, best avoided if complexity not needed, need to look up actual [syntax](http://www.tldp.org/LDP/abs/html/arrays.html))

Maybe since we invent a new convention here.

```
provides variables:
- variable_a
- ...

provides functions:
- function_a
- ...
```

That may even be something that editors could automatically assist with in future but I am thinking probably years ahead (doesn't seem too likely).